README.md

```diff
@@ -0,0 +1,74 @@
+# ReactJS Boilerplate
+
+## Roadmap
+
+Our roadmap for the ReactJS Boilerplate is where you can learn about what features we're working on.
+
+### Development environment
+
+- Build tool
+  - [x] Vite
+    - Optimisation of Minify code in production mode
+- Coding Standards and Guidelines (Statically analyzes our codes)
+  - [ ] ESLint with our rules
+  - [ ] Prettier
+- Testing framework
+  - [x] Jest
+  - [ ] React Testing Library
+  - [ ] React Hooks Testing Library
+  - [x] Storybook
+- Mock API server
+  - [x] msw
+
+### Application design
+
+- State management
+  - Client state
+    - [ ] useContext
+      - Need to have logic to prevent unnecessary rerender
+    - [ ] Use state management libraries
+      - [x] zustand
+      - Continued consideration of other proposals
+  - Server state
+    - [x] React Query
+  - Typescript rules
+    - [ ] [Interface vs Types](https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/20)
+- Routing for SPA
+  - [x] [wouter](https://github.com/molefrog/wouter)
+- Validate forms in React
+  - [x] [React Hook Form](https://react-hook-form.com)
+- CSS
+  - Preprocessors
+    - [x] Sass
+    - PostCSS
+  - Frameworks
+    - Tailwind CSS
+    - Chakra UI
+    - Material UI
+  - Animations
+
+### Application security
+
+- [Authentication](https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/15)
+- [Security](https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/9)
+
+### DX
+
+- CI / CD
+- [REST and GraphQL](https://github.com/monstar-lab-oss/reactjs-boilerplate/discussions/14)
```

Comment:
Maybe we should move REST and GraphQL to Application Design above

Reply:
DONE in https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/27/commits/5cc418d6ff7b8ea1573803dc1adc284c178dc542